### DISCUSSION TITLE
[Feature] IOS detect and update changes made to metadata

### DISCUSSION THREAD

**Discussion by Xiticks** [immich-app/immich#5413] [Feature Request]
### Idea
In IOS, using the Photos App, it's possible to add a location and/or a time to a photos (or change them), it could be great if Immich could detect theses changes (or at least having the option to force the update of a specific image) and be able to update the stored image metadata accordingly.

### What already works
From what I've tested, I can update the metadata of an image that has already been uploaded to Immich by deleting it from the server, and from the trash, then force Immich in IOS to update the library and it uploads the image again.

### Issues with what already works.
So I had an external image coming from Facebook without any metadata, I changed the date and location inside Apple Photos app, and did what I've explained above.
The photos appear inside Immich with the correct date, however there isn't any information about the localization I set. Furthermore, when I download the image from the web interface of Immich on my computer, the image doesn't contain any metadata, not even the date that has been set and I guess uploaded to Immich

### Note
I think that editing metadata inside Immich is in the pipeline, so I guess this could also solve my problem, but since I rather have a copy of my photos on my own server while also keeping them on iCloud, I'd still like to have this feature!

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by maxime-levelops** [comment]
I have the exact same use case... looking for any solution to this
